src/platform/packages/shared/kbn-esql-validation-autocomplete/src/validation/validation.test.ts

```diff
@@ -7,34 +7,33 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
+import { readFile, writeFile } from 'fs/promises';
+import { camelCase } from 'lodash';
+import capitalize from 'lodash/capitalize';
```

Comment:
nit: I understand that IDE just reorganized imports 
```suggestion
import { camelCase, capitalize } from 'lodash';
```